### ISSUE TITLE
Remove Chronic Duration dependency

### ISSUE THREAD

**Issue by Juanmcuello** [kenaniah/sidekiq-status#51]
Chronic Duration depends on [numerizer ~> 0.1.1](https://github.com/kenaniah/sidekiq-status.git), so is not possible to update to latest version ([0.2.0](https://github.com/jduff/numerizer/blob/master/lib/numerizer/version.rb#L2)).

A possible solution could be to update numerizer to 0.2.0, but last commit was 9 years ago, so I think that maybe a better approach could be to remove that dependency entirely. 

It seems to be used in the web views to express the different times intervals, but I see Sidekiq already has [some methods](https://github.com/sidekiq/sidekiq/blob/main/lib/sidekiq/monitor.rb#L123) to help with that, although not sure if all that are needed here.

I'm wondering if removing this dependency if something you are open to.

**Comment by kenaniah** [comment]
Absolutely! Feel free to open a PR with your implementation and one or more tests to confirm the functionality & prevent regressions.
